cli/lib/tasks/download.js

```diff
@@ -2,7 +2,7 @@ const arch = require('arch')
 const la = require('lazy-ass')
 const is = require('check-more-types')
 const os = require('os')
-const Url = require('url')
+const Url = require('URL')
```

Comment:
The [node documentation](https://nodejs.org/dist/latest-v16.x/docs/api/url.html#url) says this import should be:
```suggestion
const url = require('url');
```

Reply:
Done in a161000